### PR TITLE
add requireTrimmed

### PR DESCRIPTION
**Pull request by hzoo**
Add requireTrimmed into core validation rules

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
Minor nitpick: A more helpful commit message would be: `util/validate: add the requireTrimmed rule`

It gives an idea of the context of the change and a bit more clarification on what's being done.

**Comment by mrjoelkemp** [comment]
:+1:

File: test/specs/util/validate.js [hunk 1]
```diff
@@ -124,6 +124,11 @@ define(['util/validate'], function(validate) {
       expect(validate.message).toEqual('This field is required');
     });
 
+    it('verifies requireTrimmed', function() {
```
Comment (mrjoelkemp) [inline test/specs/util/validate.js]: It would strengthen the suite if you also tested that strings like `'foo '` and `' foo'` passed validation.

**Comment by jrefano** [comment]
:+1:

**Pull request merged**